srcpkgs/pywatchman/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'pywatchman'
+pkgname=pywatchman
+version=1.4.1
+revision=1
+noarch=yes
+build_style=python-module
+pycompile_module="pywatchman"
+hostmakedepends="python-devel python3-devel python-setuptools python3-setuptools"
+depends="python watchman"
+short_desc="Connect and query Watchman to discover file changes (Python 2)"
+maintainer="Hoang Nguyen <hoang@wetrust.io>"
+license="BSD"
```

Comment:
same as https://pypi.org/project/pywatchman/

Reply:
License of the project is Apache-2.0

Reply:
But of the python module is BSD-3-Clause, so use this one